drivers/net/ethernet/xilinx/xilinx_axienet_main.c

```diff
@@ -129,6 +129,8 @@ static struct axienet_option axienet_options[] = {
  */
 static inline u32 axienet_dma_in32(struct axienet_local *lp, off_t reg)
```

Comment:
instead of changing the xilinx driver, maybe it is better to create another driver.